usr/share/rear/finalize/GNU/Linux/250_migrate_disk_devices_layout.sh

```diff
@@ -32,16 +32,7 @@ local restored_file=""
 # the funny [] around the first letter make sure that shopt -s nullglob removes this file from the list if it does not exist
 # the files without a [] are mandatory, like fstab FIXME: but below there is [e]tc/fstab not etc/fstab - why?
 
```

Comment:
Those two comment lines
```
# the funny [] around the first letter ...
# the files without a [] are ...
```
should be removed here (because that is now in default.conf).

Reply:
Addressed in 8265c9c43e958309c1030270bb111e579e3089f8